source/functions.h

```diff
@@ -277,9 +277,9 @@ enum {
     F128_LT_QUIET,
 #endif
     NUM_FUNCTIONS
-};
+} function_t;
```

Comment:
Can you name this to something that is less likely to cause name collisions?`testfloat_function_t`?

Reply:
Agreed.